deployments/kustomize/backups/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ commonLabels:
   app: gdx-agreements-tracker
 # Configuration for backup
 configMapGenerator:
-  - name: pmo-backup-config
+  - name: backup-config
     literals:
       - BACKUP_DIR=/backups/
```

Comment:
I wonder if Argo CD is complaining about the trailing slash on `- BACKUP_DIR=/backups/`?

Reply:
i would look at the example form bcgov documentation to see if they add a trailing backslash